actions/index.js

```diff
@@ -122,7 +122,7 @@ export const closeStandup = async (standup) => {
     for (let i = 0; i < updates.length; i++) {
       const update = updates[i];
       const message = await formatUpdateMessage(update);
-      out.push(`${u(update.user.userId)}: ${message}`);
+      out.push(`${update.user.userId}: ${message}`);
```

Comment:
This would print the ID of the user, which isn't very helpful. Is there a way to print their name instead?

Reply:
Hmm, would I have to make another formatter and do this instead?
```
export const summaryUser = (userId) => {
  return `<${userId}>`;
};
```